config/vufind/Folio.ini

```diff
@@ -207,6 +207,15 @@ in_transit[] = "Open - Awaiting delivery"
 ; Note: `hide_holdings[]` in config.ini can be used to suppress display of specific locations.
 ; show_holdings_no_items = true
 
+; This section controls behavior on the "Checked Out Items" screen
+[Loans]
+; The size for a page of results
+page_size[] = 50
+; Sort options
+;sort[dueDate/sort.ascending] = sort_due_date_asc
+;sort[dueDate/sort.descending] = sort_due_date_desc
+;default_sort=dueDate/sort.ascending
```

Comment:
I have mixed feelings about this one. There are a few directions we could go:

1.) Uncomment some settings by default, as you suggest. This has the advantage of providing a more clear example (not to mention a potentially more comprehensible result order), but I saw some concerns on Slack that sorting can negatively impact performance, so I wonder if there's a disadvantage to having a sort active by default.

2.) We could add some logic to better validate the configuration and issue warnings if things are out of alignment (e.g. a default sort is undefined). But I'm slightly reluctant to do this because it would make the code more complicated than the current simple "pass-through to config array" setup. We could also change the higher-level code that makes use of these settings in some way, but that's beyond the scope of this PR.

3.) We could just add some clarifying comments. I'll go ahead and do that part now, since it doesn't hurt.

Reply:
I think the comment you added (direction #3) solves the issue just fine.  I'm not sure sorting performance is a real issue on this particular page, but no harm in leaving it disabled now that it's clear how to enable it.